src/test/scala/com/target/TestHelpers.scala

```diff
@@ -0,0 +1,59 @@
+package com.target.data_validator
```

Comment:
`package com.target` to correspond to location in directory

Reply:
I moved this file into `test/scala/com/target/data_validator` because `mkDict` returns a `VarSubstitution`, which is in `com.target.data_validator`

Reply:
Cleaning this up should be part of #16